apps/voting-disputable/app/src/script.js

```diff
@@ -337,6 +337,22 @@ function loadVoteData(voteId) {
   )
 }
 
+function loadVoteDisputableInfo(voteId) {
+  // Wrap with retry in case the vote is somehow not present
+  return retryEvery(() =>
```

Comment:
Genuinely curious, under which conditions would the vote not be present? while the app is syncing?

Reply:
Sorry here it should say 'Wrap with retry in case the disputable info is somehow not present'. I already update it. Its a validation in case it fails for some reason. Like for example if the event is firing a little bit before the data is available through the .call method.